package/spec/frontend/media_player/volume_binding_spec.js

```diff
@@ -103,12 +101,10 @@ describe('volumeBinding', function() {
     it('fades in after promise returned by play is resolved', function() {
       var player = fakePlayer();
       var settings = new Backbone.Model({volume: 98});
-      var deferred = new jQuery.Deferred();
-      player.play = function() { return deferred.promise(); };
+      player.play = function() { return Promise.resolve() };
       volumeBinding(player, settings);
 
       player.playAndFadeIn(500);
-      deferred.resolve();
 
       expect(player.fadingVolume).toBe(98);
```

Comment:
For all the tests that use things like `fadingVolume`/`currentVolume` we need a new strategy: Since `playAndFadeIn` is async, we can no longer observe effects directly after invocation. We also do not want to rewrite all tests to wait until the whole process is done, like we did in most of the other cases.

What we want to test is that, when we call methods like `playAndFadeIn`, `fadeVolume` is called eventually but before `playAndFadeIn` resolves. We can do this by rewriting properties like `fadingVolume` to promises as done in [this example commit](https://github.com/tf/pageflow/commit/6b7f6e62496d51fe5e60df2bc36efe1af5b5ff6c).